capsules/src/tmp006.rs

```diff
@@ -144,7 +144,8 @@ impl<'a> TMP006<'a> {
 
     fn enable_interrupts(&self) {
         // setup interrupts from the sensor
-        self.interrupt_pin.enable_input(InputMode::PullUp);
+        // TODO(alevy): do we need to make sure it's a pull up?
```

Comment:
Do you think this is something that should be in the board config (that sets up the driver) or in the TMP006 driver? In principal I suppose the board might have a physical pull-up resistor...